templates/partials/actionbar.html

```diff
@@ -62,46 +70,50 @@
-            <a class="mm-icon mm-icon-inverse mm-icon-caption mobile-only" href="sms:{{actionBar.sendTo.phone}}" ng-class="{'mm-icon-disabled': !actionBar.sendTo.phone}">
+          <div class="actions dropup" ng-show="showActionBar && actionBar.right.selected && !loadingContent && ('reports.detail' | isState)">
+            <a class="mm-icon mm-icon-inverse mm-icon-caption mobile-only" href="sms:{{actionBar.right.sendTo.phone}}" ng-class="{'mm-icon-disabled': !actionBar.right.sendTo.phone}">
               <span class="fa fa-envelope"></span>
               <p translate>Send Message</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption send-message desktop-only" data-send-to="{{actionBar.sendTo._id}}" ng-class="{'mm-icon-disabled': !actionBar.sendTo.phone}">
+            <a class="mm-icon mm-icon-inverse mm-icon-caption send-message desktop-only" data-send-to="{{actionBar.right.sendTo._id}}" ng-class="{'mm-icon-disabled': !actionBar.right.sendTo.phone}">
               <span class="fa fa-envelope"></span>
               <p translate>Send Message</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="!actionBar.verified" ng-click="emit('VerifyReport', true)">
+            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="!actionBar.right.verified" ng-click="emit('VerifyReport', true)">
               <span class="fa fa-check"></span>
               <p translate>Verify</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.verified" ng-click="emit('VerifyReport', false)">
+            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.right.verified" ng-click="emit('VerifyReport', false)">
               <span class="fa fa-times"></span>
               <p translate>Unverify</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.type === 'xml'" ui-sref="reports.edit({ reportId: actionBar.selected[0]._id })">
+            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.right.type === 'xml'" ui-sref="reports.edit({ reportId: actionBar.right.selected[0]._id })">
               <span class="fa fa-pencil"></span>
               <p translate>Edit</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.type !== 'xml'" ng-click="emit('EditReport')">
+            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-show="actionBar.right.type !== 'xml'" ng-click="emit('EditReport')">
               <span class="fa fa-pencil"></span>
               <p translate>Update Facility</p>
             </a>
-            <a class="mm-icon mm-icon-inverse mm-icon-caption" ng-click="deleteDoc(actionBar.selected)" ng-class="{'mm-icon-disabled': actionBar.disableDelete}" mm-auth="can_delete_reports">
```

Comment:
`disableDelete` was never being set. Could find no reference in the code anywhere.